src/analysis/locate.ml

```diff
@@ -1016,7 +1016,16 @@ let get_doc ~config ~env ~local_defs ~comments ~pos =
         Format.fprintf fmt "]\n"
       );
     begin match
-      Ocamldoc.associate_comment comments loc !last_location
+      let (_, deepest_before) = List.hd @@ Mbrowse.deepest_before loc.loc_start [browse] in
+      (* based on https://v2.ocaml.org/manual/doccomments.html#ss:label-comments: *)
+      let after_only = begin match deepest_before with
+        | Browse_raw.Constructor_declaration _ -> true
+        (* The remaining `true` cases are *currently* inert, since iuc we don't find
+          doc comments for these nodes yet *)
```

Comment:
Do you mean that Merlin does not show such comments or that they are not part of the specification ?
Could you add test cases showing the current behavior ?

Reply:
> Do you mean that Merlin does not show such comments or that they are not part of the specification ?

As far as I can tell, Merlin does not show such comments, at least in this codepath

> Could you add test cases showing the current behavior ?

Good idea, will do

Reply:
I added these as separate test cases here: https://github.com/ocaml/merlin/pull/1526